Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
@DirectXMan12 I have to generate deepcopies in k8s.io/metrics, but do not see any scripts for that.